### PR TITLE
DOCSP-43809: v1.10.0 Release Notes

### PR DESCRIPTION
**Pull request by mmaville-mdb**
## DESCRIPTION
Release notes for `v1.10`.

## STAGING
[Release Notes](https://deploy-preview-170--docs-relational-migrator.netlify.app/release-notes/)

## JIRA
https://jira.mongodb.org/browse/DOCSP-43809

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/release-notes.txt
Comment (sarah-olson-mongodb) [inline source/release-notes.txt:15]: (markup -- fixes the build log error)
```suggestion
----------------
```

File: source/release-notes.txt
Comment (sarah-olson-mongodb) [inline source/release-notes.txt:17]: Adding a note here to keep top of mind that the release date is still a little bit TBD.

File: source/release-notes.txt
Comment (sarah-olson-mongodb) [inline source/release-notes.txt:23]: (question)
This will resolve once that branch is merged. Is that right?

File: source/release-notes.txt
Comment (sarah-olson-mongodb) [inline source/release-notes.txt:22]: (team oxford comma)
```suggestion
  convert large numbers of SQL queries, views, and stored procedures. 
```

File: source/release-notes.txt
Comment (sarah-olson-mongodb) [inline source/release-notes.txt:26]: (minimalism) 
Let's rewrite a smidge. Something like:
- Snapshot jobs now create top-level documents using a non-idempotent bulk insert method. This improves performance.

File: source/release-notes.txt
Comment (sarah-olson-mongodb) [inline source/release-notes.txt:27]: (typo -- British spelling)
```suggestion
- Added explicit control over idempotency behavior through the 
```

Maybe also convert idempotency  to idempotent?

File: source/release-notes.txt
Comment (sarah-olson-mongodb) [inline source/release-notes.txt:31]: (question ... or somethig)
I'm pretty sure Basic Authentication doesn't need to be a proper noun, right?

File: source/release-notes.txt
Comment (sarah-olson-mongodb) [inline source/release-notes.txt:38]: (tense)
You could change to were, if you like?

**Review by sarah-olson-mongodb — COMMENTED** [review]
Thanks a ton @mmaville-mdb! This is a great start. Couple items for your consideration. Happy to take another super quick look! 

Gentle reminder to not copy/paste, even for release notes. :)

File: source/release-notes.txt
Comment (mmaville-mdb) [reply source/release-notes.txt:23]: That's correct! I just merged the bulk conversions working PR into the feature branch.

File: source/release-notes.txt
Comment (mmaville-mdb) [reply source/release-notes.txt:30]: I'm going to tag @tomhollander for input on this one. I believe in this context, "Basic" might be the only proper noun if we're going by the RFC. I've also seen "Basic HTTP authentication" thrown around.

**Review by tomhollander — APPROVED** [review]
LGTM but I think @davidsoulmdb may have another item to add.

**Comment by davidsoulmdb** [comment]
Additional change: For the "Snapshot jobs now create top-level documents using a non-idempotent bulk insert method. This improves performance." I would mention that "Snapshot jobs now create top-level documents using a non-idempotent bulk insert method, **and internal indexes used for migration are now automatically deleted after job completion.** This improves performance"

**Comment by davidsoulmdb** [comment]
I have an ask above to please also mention that we delete some internal indexes as part of our performance improvements

File: source/release-notes.txt
Comment (mmaville-mdb) [reply source/release-notes.txt:32]: Taggin @davidsoulmdb for this one as well.

Comment (sarah-olson-mongodb) [reply source/release-notes.txt:32]: My bad. I googled and see that Basic Authentication is sometimes used as a proper noun.

**Review by sarah-olson-mongodb — APPROVED** [review]
LGTM and thanks a ton!

**Comment by alwaysbemark** [comment]
Hey folks - only thing I'd add (still needs @davidsoulmdb 's approval) - we should treat this as the 1.10.0 version (with the .0 bugfix suffix) - this is important because if a critical bug is found we'll be releasing 1.10.1 etc, so they need a way to be represented as well.

**Comment by davidsoulmdb** [comment]
LGTM

**Review by davidsoulmdb — APPROVED** [review]
Ready to post tomorrow, will confirm over slack channel

**Pull request merged**
